### PR TITLE
refactor: change the way entered AccNum and AccIndex to use config.toml

### LINKED ISSUE #61: Change the way input of account number and index received
Currently, the account number and index for the oracle account are entered as flags.
Change to receive input from `config.toml`.

### PR DESCRIPTION
**Pull request by 0xHansLee**
Close #61

### CODE REVIEW

File: config/toml.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ const DefaultConfigTemplate = `# This is a TOML config file.
 
 log-level = "{{ .BaseConfig.LogLevel }}"
 oracle-mnemonic = "{{ .BaseConfig.OracleMnemonic }}"
+acc-num = "{{ .BaseConfig.AccNum }}"
+acc-index = "{{ .BaseConfig.AccIndex }}"
```
Comment (gyuguen) [inline config/toml.go]: For unity, how about prefixing oracle like this:
```suggestion
oracle-acc-num = "{{ .BaseConfig.AccNum }}"
oracle-acc-index = "{{ .BaseConfig.AccIndex }}"
```

**Review by gyuguen — APPROVED** [review]
LGTM 👍 
I left a non-important comment.

File: config/toml.go (hunk 1 above)
Comment (0xHansLee) [reply config/toml.go]: Good. agree with you. I'll change it

**Review by youngjoon-lee — APPROVED** [review]
good

**Review by inchori — APPROVED** [review]
LGTM 👍

**Pull request merged**